eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -73,7 +73,7 @@ public function loadBinaryFile($binaryFileId)
     {
         $this->checkBinaryFileId($binaryFileId);
 
-        if ($binaryFileId[0] === '/') {
+        if ($binaryFileId[0] === '/' || (PHP_OS === 'WINNT' && $binaryFileId[1] === ':')) {
```

Comment:
Above, I have not the cleanest way of checking if a given path is an absolute one. On Linux colon is a valid character in a path (though unreasonable). I can skip it and use single RegEx for any platform, but maybe we should introduce a trait and rather do: `$this->isAbsolutePath($binaryFileId)`. WDYT @andrerom?